app/lib/notifications.server.ts

```diff
@@ -34,14 +35,16 @@ const sendNotification = async (emailAddress: string, subject: string, text: str
   try {
     const transport = initializeTransport();
     logger.debug(`Sending notification to ${emailAddress}`);
-    return await transport.sendMail({
+    await transport.sendMail({
       from: NOTIFICATIONS_EMAIL_USER,
       to: emailAddress,
       subject,
       text,
     });
+    return true;
```

Comment:
I don't think this is right.  If it works, return nothing.  If it fails, throw.

You should return the promise, and in tests you can expect it to resolve or reject.